project/sites/ppsni/themes/ppsni_theme/src/scss/components.scss

```diff
@@ -145,3 +168,22 @@
     }
   }
 }
+
+.region-footer {
+  background: #212129;
```

Comment:
Unless the task says so I don't think we change the footer colour from what the default Unity footer colours are. We normally change the border-top but this is done through a variable in `init/_colors.scss`. In there there are 4 variables for the footer.
```
$footer-background: color(footer-background);

$footer-border: color(brand-primary);

$footer-text: color(footer-text);

$footer-headings: color(footer-headings);
```

Reply:
Done, thanks Adam